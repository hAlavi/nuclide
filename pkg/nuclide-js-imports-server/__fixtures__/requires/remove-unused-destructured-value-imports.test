import {Y, Z} from 'A';
import {m, n} from 'B';

const y = Z + m;
