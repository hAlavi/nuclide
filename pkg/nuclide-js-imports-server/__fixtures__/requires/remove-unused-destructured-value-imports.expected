import {Z} from 'A';
import {m} from 'B';

const y = Z + m;
